execution_workflows/APA-Scan/config/samples.csv

```diff
@@ -0,0 +1,3 @@
+sample,sample_type,data_type
```

Comment:
Not sure how, but I think run-/sample-specific configs should be organized in some way to make it easy to run specific sets of matching configs (with each run relating to a parameter code in the list we prepared). It's okay to keep this as a dummy example though I think